src/msg/async/ProtocolV1.cc

```diff
@@ -726,6 +730,29 @@ CtPtr ProtocolV1::throttle_dispatch_queue() {
           << connection->dispatch_queue->dispatch_throttler.get_current() << "/"
           << connection->dispatch_queue->dispatch_throttler.get_max()
           << " failed, just wait." << dendl;
+      std::chrono::seconds configured_interval = msgr->dispatch_throttle_log_interval.load();
+      ceph::coarse_mono_time throttle_now = ceph::coarse_mono_clock::now();
+      if (configured_interval.count()) {
+        if (std::chrono::duration_cast<std::chrono::seconds>(throttle_now - throttle_prev) >=
+            configured_interval) {
+          ldout(cct, 1) << __func__ << " Throttler Limit has been hit. "
+                        << "Some message processing may be significantly delayed." << dendl;
+          throttle_prev = throttle_now;
+        }
+      }
+      configured_interval = msgr->dispatch_throttle_clog_interval.load();
```

Comment:
The name of this config is weird. What it really controls is how frequently `ms_deliver_throttle` is called, no?